README.md

```diff
@@ -1,5 +1,74 @@
-# The National Archivs cookie banner consent
+# Cookie banner consent Wordpress plugin
 
-v1.0 -- 2021-03-30
+- [Project findings](https://github.com/nationalarchives/ds-cookie-consent/blob/mihai/wp_plugin/docs/about-the-project.md)
 
-- [Project findings](https://github.com/nationalarchives/ds-cookie-consent/blob/main/docs/about-the-project.md)
+## Installation
+- Upload, install and activate the plugin
+
+## Create Cookies Settings page
+
+- Add shortcode [ds-cookie-consent-settings] to the website
+
+- Content for the cookie sections ca be added from TNA Cookie Consent dashboard settings page
```

Comment:
I think there's a typo here: `ca` instead of `can`

Reply:
Done